detox/test/e2e/25.statusbar.test.js

```diff
@@ -0,0 +1,170 @@
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+describe('StatusBar Override', () => {
+  it('Time', async () => {
+    await device.setStatusBar({
+      time: '11:11',
+    });
+    await sleep(500);
+    await device.setStatusBar({
+      time: '22:22',
+    });
+    await sleep(500);
+    await device.setStatusBar({
+      time: '33:33',
+    });
+    await sleep(500);
+    await device.resetStatusBar();
```

Comment:
If there is no way to assert that the commands have an effect on the simulator status bar, could you at least ensure that each test runs a single command? So that `beforeEach` and `afterEach` screenshots will be demonstrating the difference.

Reply:
Move `await device.resetStatusBar();` in `beforeEach()` scope.

Reply:
Most likely, you'll have to use `await device.takeScreenshot()` API to yield two different screenshots in this case.

Reply:
Fixed

Reply:
Is there a reason why you have removed `sleep(500)` now in e2e tests?

Reply:
The `sleep(500)` was to visually follow the test.
As we have the screenshots now, there's no reason to sleep as we can check the screenshots.

But I can place it back if you want.